### DISCUSSION TITLE
Testing on TV Box RK3318 H96 MAX Android 11

### DISCUSSION THREAD

**Discussion by ghost** [DesktopECHO/Pi-hole-for-Android#20] [Show and tell]
[SCREEN FETCH + PI-HOLE](https://ibb.co/M6BkBHW)

I would like to run the pi-hole on a dedicated device, but here I have only one TV box available, "full time on".
Uninstalling android to perform the server function through armbian headless would be a waste of an integrated GPU.
so i found your project that came in handy with my need and the need of several homes that don't have a dedicated server even though raspian.

03/10/2022 Status: 
Testing on [rooted device](https://www.youtube.com/watch?v=AkwPjJN5fUI).

03/10/2022 Finish deploy: 
I used the internal memory for the disk image.
Just sent the router ipv6 + ipv4 traffic over the TV box,
no adjustments were needed, everything works out of the box

04/10/2022 Daily Usage:
i dont recomend you to use the Teleport Function from Pi-hole.
Client, Group, Audit log, Static DHCP Leases, Local DNS Records
it cause the system to crash.
i have problems with android time zone desync
Pihole + Unbound - SERVFAIL for all queries
be careful to enable your dns fully by android device, 
if he doesn't have the realtime battery
make at least one time zone server by your main router

**Comment by DesktopECHO** [comment]
Hi, did the solution in https://github.com/DesktopECHO/Pi-hole-for-Android/issues/21 correct the time/date sync issue?

Also, can you retry the Teleport functionality or list the steps to reproduce the issue?  I have not run into this problem before.

**Reply by ghost** [reply]
i will test the #21  soon, Waiting to comercial time finish here, I already installed[ ntp on this device](https://pplware.sapo.pt/linux/ntp-sincronize-o-relogio-do-seu-linux-centos-7/) it's up and runing.
The teleport function add Users to pi-hole 
i think this distro contains a free access to admin page without login
so, when i bring the pi-hole-Debian-Hole-teleporter.tar.gz File from OLD instalation,
it brings the[ user table and admin login data](https://ibb.co/Gx6hxvH),
this isn't a problem, because[ i can select the items that i can bring from old instalations.](https://ibb.co/TWT5shC)

**Reply by DesktopECHO** [reply]
FYI, just released v1.7 with time-sync and other improvements.

**Reply by ghost** [reply]
> FYI, just released v1.7 with time-sync and other improvements.

Testing NOW

**Comment by ghost** [comment]
[V1.7 no problems at all](https://ibb.co/F30rRPQ)

**Reply by DesktopECHO** [reply]
Great news, thanks for the report!
D.
